cypress/integration/buy-shirt.spec.ts

```diff
@@ -0,0 +1,27 @@
+
+describe("Buy a t-shirt", () => {
+
+    it("then the t-shirt should be bought", () => {
+      cy.visit("http://automationpractice.com/")
+      cy.get("#block_top_menu > ul > li:nth-child(3) > a").click()
+      cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click()
+      cy.get("[style*=\"display: block;\"] .button-container > a").click()
+      cy.get(".cart_navigation span").click()
+  
+      cy.get("#email").type("aperdomobo@gmail.com")
+      cy.get("#passwd").type("WorkshopProtractor")
+  
+      // Debes completar la prueba ...
+      
+      cy.get("[name=SubmitLogin]").click()
+      cy.get("[name=processAddress]").click()
+      cy.get("[name=cgv]").click()
+      cy.get("[name=processCarrier]").click()
+      cy.get("[class=bankwire]").click()
+      cy.get("#cart_navigation > button").click()
+
+      cy.get("#center_column > div > p > strong")
+        .should("have.text", "Your order on My Store is complete.")
+    });
+  });
+  
```

Comment:
add new line at the end of the file

Reply:
line added.

Reply:
please make sure that the last line is empty, it should not contain any space :)

Reply:
When GitHub shows the following icon:
![image](https://user-images.githubusercontent.com/10209180/126735042-e795545a-4a49-4fed-83f6-5a1b0aa8a24d.png)

Means that the file does not end with just one and only one empty or blank line